### PR TITLE
Beautiful Math - Remove title attribute from module generated image

### LINKED ISSUE #18954: Math Latex Module - Title Attribute
#### Steps to reproduce the issue

1. Enable the 'Beautiful Math' module
2. Add an example Latex equation to an HTML block. Example equation is pasted here below.

```
$latex
\text{Recognize that this is a separable differential equation.} \\
\\
\begin{aligned}
\dfrac{dy}{dt} &amp;= -y \cos(t) &amp;&amp; \text{Subtract } y \cos(t) \text{ from both sides.} \\
dy &amp;= -y \cos(t) dt  &amp;&amp; \text{Multipy by } dt \text{.} \\
\dfrac{dy}{y} &amp;= \cos(t) dt &amp;&amp; \text{Divide by } y \text{ on both sides.} \\
\int \dfrac{dy}{y} &amp;= \int \cos(t) dt &amp;&amp; \text{Prepare to integrate.} \\
\ln |y| &amp;= \sin(t) + C \mspace{100.0mu} &amp;&amp; \text{Integrate both sides.} \\
y &amp;= e^{\sin(t) + C} &amp;&amp; \text{Rewrite in exponential form.} \\
y &amp;= e^{\sin(t)} \cdot e^{C} &amp;&amp; \text{Rewrite as the multiplication of two common bases.} \\
y &amp;= Ce^{\sin(t)} &amp;&amp; \text{Let } e^{C}=C \text{.} \\
\\
\text{Answer: } \boxed{y = Ce^{\sin(t)}}
\end{aligned} $
```

3. Publish page
4. On the published page, if you hover over the generated image, the title will show in a tooltip with the same LaTex input that was entered into the block.

latex-jetpack

#### What I expected
I would expect to not see any title and if one is showing, one that is more relevant than this LaTex text.

#### What happened instead
I believe that we could just remove the title from this image on line 113 in the file below. 

https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/modules/latex.php

I am not an SEO expert, but it does not seem the Title in this instance is relevant or aiding in describing what the image is.

https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/

### PR DESCRIPTION
**Pull request by joweber123**
Removing the title tag from the generated image.  

Fixes #18954

#### Changes proposed in this Pull Request:
* This is a change to remove the title tag from the image this module generates.  

#### Jetpack product discussion
This was first noted here 3777668-zen

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Add sample LaTex code to a page/post.  Look at the HTML of the generated image.  The title attribute should be removed. 

#### Proposed changelog entry for your changes:
Remove title attribute from Beautiful Math generated LaTex image. 

#### Other Info
This is my first pull request so please let me know how better I can test this out!

### CODE REVIEW

**Review by jeherve — APPROVED** [review]

**Comment by joweber123** [comment]
@jeherve I see that step 10 [here](https://github.com/Automattic/jetpack/blob/master/docs/pull-request.md) is "The release manager will take a look and test your branch, and merge it into master so it can be part of the next release."

Is that still accurate?  Anything else I need to do here other than wait for the release manager?  

(This is my first time working on a Jetpack issue so just wanted to verify)

**Comment by jeherve** [comment]
> Is that still accurate? Anything else I need to do here other than wait for the release manager?

You don't have to do anything else, I'll merge that PR for you. :)

**Comment by jeherve** [comment]
r222029-wpcom

**Pull request merged**
